### PR TITLE
Add my profile to AboutUs.md

### PR DESCRIPTION
**Pull request by kuanjr**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -9,15 +9,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+### Kuan Jian Rong
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jr.png" width="200px">
+
+[[github](https://github.com/kuanjr)]
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
```
Comment (j4ck990) [inline docs/AboutUs.md:18]: Perhaps you need to leave this link and have a skeleton project portfolio page

File: docs/AboutUs.md [hunk 2]
```diff
@@ -9,15 +9,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Kuan Jian Rong
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jr.png" width="200px">
```
Comment (j4ck990) [inline docs/AboutUs.md:14]: What a handsome young man.

**Review by j4ck990 — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (AD-NAP) [reply docs/AboutUs.md:18]: I agree with Jack, i think best to link the github to your own github page and portfolio to your-github-username.md in the team folder.

File: docs/AboutUs.md (hunk 2 above)
Comment (AD-NAP) [reply docs/AboutUs.md:14]: Perhaps the filename of the profile photo should be kuanjr.png as mentioned in the textbook to name it to github_username_in_lower_case.png

**Review by j4ck990 — APPROVED** [review]

**Pull request merged**